product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md

```diff
@@ -15,6 +17,14 @@ Das _fiskaltrust.Middleware.SCU.DE.Epson_-Package ermöglicht die Verbindung der
 | _deviceid_ | Die für die TSE spezifische Geräte-ID. Falls nicht angegeben, wird der Defaultwert verwendet. | ja |
```

Comment:
Don´t know if this section is still necessary?

Reply:
i also don't know, there i'll leave it in for now